src/ray/protobuf/gcs.proto

```diff
@@ -97,18 +97,16 @@ message ActorTableData {
   bytes actor_creation_dummy_object_id = 3;
   // The ID of the job that created the actor.
   bytes job_id = 4;
-  // The ID of the node manager that created the actor.
-  bytes node_manager_id = 5;
```

Comment:
Why not retain this separately?

Reply:
I think it is nice to think of the node ID as part of the address to simplify failure handling when sending messages. Whoever has an address should try the IP and port, but if the node has been marked as dead, then it can assume that the other end has failed.

Reply:
Makes sense. Add a note in the proto?